modules/ROOT/pages/index-self-managed.adoc

```diff
@@ -1,26 +1,76 @@
 
-image::architecture-self-managed.png[]
+Linux-based operating systems are required for all nodes that run Runtime Fabric components on Self-Managed Kubernetes clusters.
 
-MuleSoft is responsible for providing support for Runtime Fabric services, the Mule Runtime server, and Mule applications. You are responsible for creating, configuring and managing the Amazon EKS, AKS, or GKE environment.
+=== The Runtime Fabric Agent
 
-== Shared Responsibility
+The Runtime Fabric agent is deployed as a pod in the cluster and communicates with the control plane via an mTLS outbound connection established at startup. 
 
-The successful operation of Anypoint Runtime Fabric on Self-Managed Kubernetes is a shared responsibility. It is critical to understand which areas you must manage and which areas are managed by MuleSoft.
+The agent is event-driven. When a Kubernetes event is generated by an application state change, the agent sends metadata to the control plane describing the current state of the application. Events include application pod starts, updates, or restarts.
+
+The agent also listens for incoming requests from the control plane. When the agent receives a message from the control plane, the agent makes changes to the appropriate Kubernetes resources as specified in the message. Such changes include creating a new application, updating an existing application, or deleting an application. 
+
+=== Additional Cluster Components
+
+The following Runtime Fabric Components also run inside the Kubernetes cluster: 
+
+* Mule Cluster IP Service: Provides an API that Mule applications use to discover their peers inside the cluster
+* Resource cache: Provides a cluster-local cache of application dependencies
+* Persistence Gateway: Provides a persistent ObjectStore interface to Mule applications
+
+=== About Component Installations and Upgrades
+
+When you install Runtime Fabric on your Kubernetes cluster, the Runtime Fabric command-line utlity (`rtfctl`) swaps an activation data token for a set of activation properties. Properties include credentials to access the Runtime Fabric Docker Registry and credentials to establish an mTLS connection to the Anypoint platform. It then invokes Helm to perform the installation. 
+
+You trigger upgrades to the Runtime Fabric components in your cluster from the Anypoint control plane, specifically from Runtime Manager. When you trigger an upgrade, an upgrade message is processed inside the target cluster, and `rtfctl` invokes Helm to upgrade the components' software versions. If you encounter a problem during an upgrade, you can use `rtfctl` to invoke an Helm rollback and restore the cluster to a stable state. A rollback creates a report in the control plane.
+
+== How Application Deployments Work in Runtime Fabric on Self-Managed Kubernetes 
+
+When you deploy an application in Runtime Fabric on Self-Managed Kubernetes, the following occurs:
```

Comment:
```suggestion
When you deploy an application in Anypoint Runtime Fabric on Self-Managed Kubernetes, the following occurs:
```

Reply:
Again, here and below, I believe that the guidance in the naming doc is to:

- Abbreviate only Anypoint Runtime Fabric to Runtime Fabric
- Abbreviate ARFoSMK to one of the three options, or not abbreviate it at all

Checking with my MarComm contact now....

Reply:
CONFIRMED